CHANGELOG.md

```diff
@@ -10,11 +10,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Added
 
 - `K8s.Server.AsyncStreamRunner` to run streams in a separate process
+- `K8s.Sys.Telemetry` defines `telemetry` spans and events
 
 ## Updated
 
 - `K8s.Server.Watcher` and `K8s.Server.Reconciler` were rewritten completely. They now prepare streams which are to be run with `K8s.Server.AsyncStreamRunner`
 
+## Removed
+
+- `K8s.Sys.Event` was removed in favor of `K8s.Sys.Telemetry`
```

Comment:
Why rename this? It would not be possible to just change the underlying implementation and keep the existing contract, this makes life difficult for the user who is trying to update the version. The user doesn't need to know about implementation details, plus Event makes much more sense in Kubernetes terminology

Reply:
Also thanks @sleipnir for the review and feel free do more of it. :)

Reply:
Thanks @mruoss